nginx/README.md

```diff
@@ -190,11 +199,12 @@ For containerized environments, see the [Autodiscovery Integration Templates][8]
 
 **Note**: This `<INSTANCE_CONFIG>` configuration works only with NGINX Open Source. If you are using NGINX Plus, inline the corresponding instance configuration.
 
-##### Log collection
+
+#### Log collection
```

Comment:
There is one less # here, is it right?

Reply:
yes, this is how the templates do it, not sure which one is correct. https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/docs/setup/configuration/log_collection.yaml